changelog/5673.improvement.md

```diff
@@ -0,0 +1,88 @@
+Expose diagnostic data for action and NLU predictions.
```

Comment:
Can we do this in a separate PR, please?

Reply:
Besides, nobody is supposed to use that outside of RasaLit.

Reply:
> Besides, nobody is supposed to use that outside of RasaLit.

Why not? Maybe the want to build a similar tool. I don't think we should have undocumented features in our code. I think CSE should call the shots as they're owning the docs and we usually never merge anything to `master` without documentation.

Reply:
I'm  not sure I understand this well enough to give a super informed idea of where it should go. What I would recommend is adding it on the Tuning Your Model page. 

Right now there is an H2 for "Handling Class Imbalance"; I think we could turn this into an H2 for something like "Fine-Tuning Your model", so:

How to Choose a Pipeline
  ...
Choosing the Right Components 
  ... 
Fine-Tuning  <maybe there is a more general term for this that isn't an actual ML concept>
  Handling Class Imbalance
  Understanding Diagnostic Data
Configuring Tensorflow



then you'd just want a quick paragraph or 2 with basic info like
- how to access the information
- how to interpret it, on a general level, or what sorts of issues it can help you tackle

Reply:
How about adding a new section `Accessing prediction weights` and then put the content of the changelog there?

Reply:
Did that now, but named it "Accessing Diagnostic Data"